Library/test-helpers/lib.sh

```diff
@@ -1716,6 +1716,238 @@ limeSubmitCommonLogs() {
 
 }
 
+true <<'=cut'
+=pod
+
+=head2 limeconCreateNetwork
+
+Create container network.
+
+    limeconCreateNetwork NAME SUBNET
+
+=over
+
+=item SUBNET
```

Comment:
Nitpick but could you document arguments in the same order as they are being used above?